### DISCUSSION TITLE
Why limit it to Documents and Photo only?

### DISCUSSION THREAD

**Discussion by lamyergeier** [TagStudioDev/TagStudio#267] [Ideas]
I would like to also tag 
- music files
- GPX (map) files
- Videos
- URL bookmarks

Each in separate library. 

## Observation

I tried to create a new library at a location with about 100 video files. It took a while to load all files and the software was so slow and it kept show "Force close as not responding". May be we really need CLI version of the app.

After the library loaded completely with all thumbnails of the video. I closed it and again opened the library, it again took similar amount of time to load the library!

**Comment by CyanVoxel** [comment]
The program is not limited to documents and photos - you can tag any available files in your directories, including the file types you've listed. I've also addressed your observation in #265.
